unittests/snapshot_tests.cpp

```diff
@@ -424,7 +424,6 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_compatible_versions, SNAPSHOT_SUITE, snapshot
    std::string current_version = "v6";
 
 #warning update test for v7
```

Comment:
Update the `for` loop below and the `current_version`. Make sure there is a test that verifies that v7 can read prior formats. Remove warning.

Reply:
done